plonk/src/circuit/customized/mod.rs

```diff
@@ -964,7 +964,7 @@ pub(crate) mod test {
     fn build_is_zero_circuit<F: PrimeField>(a: F) -> Result<PlonkCircuit<F>, PlonkError> {
```

Comment:
we should change these test fn names as well.

Reply:
nice catch. Fixed.